handbook/engineering/web/extensibility/index.md

```diff
@@ -57,19 +64,16 @@ We are not planning on growing the Extensibility team further in 2021.
 ## Processes
 
 ### Backlog
+We use backlogs for tracking bugs, small features, and unplanned work. You can find the [extensibility backlog](https://github.com/orgs/sourcegraph/projects/116?fullscreen=true) on Github. Please note that, we don't use the backlog for tracking work that is expressly planned in our [roadmap](https://sourcegraph.productboard.com/roadmap/2599047-extensibility-fy22-roadmap). 
 
-The team keeps a backlog GitHub project board: [extensibility backlog](https://github.com/orgs/sourcegraph/projects/116). 
-
-We use the backlogs for tracking bugs, small features, and unplanned work. We don't use the backlog for tracking work that is expressly planned in our [roadmap](goals.md#roadmap). 
-
-To add an issue, tag it `team/extensibility` to notify the Extensibility team and put it in the "to triage" column of the board. Unless you're directly asked, only extensibility team members should move issues out of the "to triage" column on the board. A team member will confirm the issue is team-related, and then move it to the appropriate column.
+To add an issue, tag it `team/extensibility` to notify the Extensibility team and put it in the "to triage" column of the board. Unless you're directly asked, only extensibility team members should move issues out of the "to triage" column on the board. A team member will confirm the issue is team-related and then move it to the appropriate column.
 
 #### Tracking Issues
 The team makes use of [tracking issues](../../tracking_issues.md) for tracking progress on the implementation of new features. The teammates should ensure that a tracking issue is created when starting work on features that are expected to take longer than a single iteration to deliver.
 
 #### Product Feedback
 
-Specific product feedback about well-defined, small features can be found directly in the backlog boards. More general product feedback that applies to larger features, or that needs more research and planning to be actionable, is kept in [Productboard](https://sourcegraph.productboard.com/feature-board/2330167-web-extensibility)
+Specific product feedback about well-defined, small features can be found directly in the backlog boards. More general product feedback that applies to larger features, or that needs more research and planning to be actionable is kept in [Productboard](https://sourcegraph.productboard.com/feature-board/2330167-web-extensibility)
```

Comment:
```suggestion
Specific product feedback about well-defined, small features can be found directly in the backlog boards. More general product feedback that applies to larger features or that needs more research and planning to be actionable is kept in [Productboard](https://sourcegraph.productboard.com/feature-board/2330167-web-extensibility)
```
Nit: grammar's a bummer haha but if we want to remove the pause entirely I believe both get cut (had commas for clarity of the hierarchy of or/and but that's a stylistic choice I think)